content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 50
```

Comment:
I am not sure how the website caching responds to using the same weight for a previously weighted item.  @geeknoid is using the same weight as Service Graph (which is deprecated) ok here?

Reply:
Perhaps I should make this 49 instead - just one less of the deprecated ServiceGraph, since its meant to take its place later. This would still leave the forties (40-48) available. We can bump this back up to 50 when ServiceGraph Task is removed in the ~1.2 timeframe.